### PR TITLE
feat: support _callback and callback

### PR DESCRIPTION
**Pull request by dead-horse**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ exports.jsonp = {
 
 ## Configuration
 
-* {String} callback - jsonp callback method key, default to `_callback`
+* {String|Array} callback - jsonp callback method key, default to `[ '_callback', 'callback' ]`
```
Comment (dead-horse) [inline README.md:43]: 基本上所有的 jsonp 请求库默认都是用的 callback，还是兼容一下，否则用起来比较麻烦

File: package.json [hunk 2]
```diff
@@ -23,15 +23,15 @@
...
-    "webstorm-disable-index": "^1.1.2"
+    "autod": "^2.8.0",
+    "egg": "^1.4.0",
+    "egg-bin": "^3.4.1",
+    "egg-ci": "^1.6.0",
+    "egg-mock": "^3.7.1",
+    "eslint": "^3.19.0",
+    "eslint-config-egg": "^4.2.0",
+    "supertest": "^3.0.0",
+    "webstorm-disable-index": "^1.2.0"
```
Comment (atian25) [inline package.json:34]: 顺便 `"version": "6, 8"`

**Review by atian25 — APPROVED** [review]

**Comment by atian25** [comment]
7 没必要了吧

**Review by fengmk2 — APPROVED** [review]

**Comment by fengmk2** [comment]
我发个 minor

**Comment by fengmk2** [comment]
1.1.0

**Comment by dead-horse** [comment]
> 7 没必要了吧

先测着，等不维护了再去掉

**Pull request merged**